### DISCUSSION TITLE
Valheim server wont start

### DISCUSSION THREAD

**Discussion by Empulakcz** [GameServerManagers/LinuxGSM#4074] [Q&A] [answered]
Hello,

may i ask you for help regarding Valheim server hosted on debian?

The server is freshly installed today. 

Please check my configuration https://termbin.com/xgnw

When did a run for firstime i received information server started OK. But after running dt command its in STOPPED state. When i run start command again i receive FAIL error. 

In console log i can see:

linuxgsm@debian:~$ cat log/console/vhserver-console.log
Mono path[0] = '/home/linuxgsm/serverfiles/valheim_server_Data/Managed'
Mono config path = '/home/linuxgsm/serverfiles/valheim_server_Data/MonoBleedingEdge/etc'
Preloaded 'libsteam_api.so'
Unable to preload the following plugins:
        libparty.so
Initialize engine version: 2020.3.33f1 (915a7af8b0d5)
[Subsystems] Discovering subsystems at path /home/linuxgsm/serverfiles/valheim_server_Data/UnitySubsystems
Forcing GfxDevice: Null
GfxDevice: creating device client; threaded=0
NullGfxDevice:
    Version:  NULL 1.0 [1.0]
    Renderer: Null Device
    Vendor:   Unity Technologies
Begin MonoManager ReloadAssembly
- Completed reload, in  0.094 seconds
ERROR: Shader Sprites/Default shader is not supported on this GPU (none of subshaders/fallbacks are suitable)
ERROR: Shader Sprites/Mask shader is not supported on this GPU (none of subshaders/fallbacks are suitable)
ERROR: Shader GUI/Text Shader shader is not supported on this GPU (none of subshaders/fallbacks are suitable)
UnloadTime: 0.688100 ms
Caught fatal signal - signo:11 code:1 errno:0 addr:0x68
Obtained 49 stack frames.
#0  0x007f9ba6cc1730 in funlockfile
#1  0x007f9ba8d79d3d in _dl_rtld_di_serinfo
#2  0x007f9ba8d811ae in _dl_find_dso_for_object
#3  0x007f9ba6c238cf in _dl_catch_exception
#4  0x007f9ba8d80bca in _dl_find_dso_for_object
#5  0x007f9ba6aeb256 in (Unknown)
#6  0x007f9ba6c238cf in _dl_catch_exception
#7  0x007f9ba6c2395f in _dl_catch_error
#8  0x007f9ba6aeb995 in dlerror
#9  0x007f9ba6aeb2e6 in dlopen
#10 0x007f9ba657b28e in mono_dl_open
#11 0x007f9ba64f6ecd in mono_jit_info_get_method
#12 0x007f9ba64f8017 in mono_lookup_pinvoke_call
#13 0x007f9ba650af0c in mono_marshal_string_to_utf16
#14 0x007f9ba63cba82 in mono_set_break_policy
#15 0x007f9ba63b04a4 in _init
#16 0x007f9ba63b1f1c in _init
#17 0x007f9ba63b5dfa in mono_print_method_from_ip
#18 0x007f9ba6435bde in mono_print_thread_dump_from_ctx
#19 0x007f9ba6435f0d in mono_print_thread_dump_from_ctx
#20 0x00000041105393 in (Unknown)
#21 0x00000040717d68 in System.TimeZoneInfo:TryGetNameFromPath (string,string&)
#22 0x0000004070dd18 in System.TimeZoneInfo:get_Local ()
#23 0x007f9ba63b6cdd in mono_print_method_from_ip
#24 0x007f9ba6526b18 in mono_perfcounter_foreach
#25 0x007f9ba65279c5 in mono_runtime_invoke
#26 0x007f9ba7d3ab62 in scripting_method_invoke(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingArguments&, ScriptingExceptionPtr*, bool)
#27 0x007f9ba7d38c72 in ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool)
#28 0x007f9ba7d18e47 in Coroutine::InvokeMoveNext(ScriptingExceptionPtr*)
#29 0x007f9ba7d18a75 in Coroutine::Run(bool*)
#30 0x007f9ba7d1c1de in MonoBehaviour::TryCreateAndRunCoroutine(ScriptingObjectPtr, ScriptingMethodPtr, Coroutine**)
#31 0x007f9ba7d1c5b2 in MonoBehaviour::StartCoroutineManaged2(ScriptingObjectPtr)
#32 0x007f9ba7e0bffe in MonoBehaviour_CUSTOM_StartCoroutineManaged2(ScriptingBackendNativeObjectPtrOpaque*, ScriptingBackendNativeObjectPtrOpaque*)
#33 0x0000004070d16e in (wrapper managed-to-native) UnityEngine.MonoBehaviour:StartCoroutineManaged2 (UnityEngine.MonoBehaviour,System.Collections.IEnumerator)
#34 0x007f9ba63b6cdd in mono_print_method_from_ip
#35 0x007f9ba6526b18 in mono_perfcounter_foreach
#36 0x007f9ba65279c5 in mono_runtime_invoke
#37 0x007f9ba7d3ab62 in scripting_method_invoke(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingArguments&, ScriptingExceptionPtr*, bool)
#38 0x007f9ba7d38c72 in ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool)
#39 0x007f9ba7d1d201 in MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingExceptionPtr*)
#40 0x007f9ba7d1d322 in MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr)
#41 0x007f9ba7d1dbf6 in MonoBehaviour::DelayedStartCall(Object*, void*)
#42 0x007f9ba7a54202 in DelayedCallManager::Update(int)
#43 0x007f9ba7bba957 in ExecutePlayerLoop(NativePlayerLoopSystem*)
#44 0x007f9ba7bba912 in ExecutePlayerLoop(NativePlayerLoopSystem*)
#45 0x007f9ba7bbaebe in PlayerLoop()
#46 0x007f9ba7d98bab in PlayerMain(int, char**)
#47 0x007f9ba6b1309b in __libc_start_main
#48 0x0055589e463029 in _start

Any idea whats going on please?

**Comment by Empulakcz** [comment, marked as answer]
Hello, 

i´ve just figured it out. Its probably problem i use older Debian 10 Buster and its related to new crossplay function which has been added recently.

I found out that plugin Libparty.so which is located at vhserver/serverfiles/valheim_server_Data/Plugins/libparty.so is causing the issue. So i just renamed this file and server has started with no issue. 

Note: People with newer version of Debian probably would not have this issue.

**Reply by JHikyu** [reply]
Thanks.
